### PR TITLE
Apple Pay additional config

### PR DESCRIPTION
**Pull request by BushraAsif**

### CODE REVIEW

**Review by shahbaz-mehar — COMMENTED** [review]
Changes have been reviewed.

File: etc/adminhtml/system/terminal5.xml [hunk 1]
```diff
@@ -33,6 +33,10 @@
             <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
             <config_path>payment/terminal5/isapplepay</config_path>
         </field>
+        <field id="applepaylabel" translate="label comment" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
```
Comment (emicha) [inline etc/adminhtml/system/terminal5.xml:36]: Could you add a comment for what the label is being used for?

Comment (emicha) [inline etc/adminhtml/system/terminal5.xml:36]: Could you add a comment for what the label is being used for?

Comment (BushraAsif) [reply etc/adminhtml/system/terminal5.xml:36]: Added

**Review by emicha — APPROVED** [review]

**Pull request merged**